newrelic-notify.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -ex pipefail
+
+NEWRELIC_USER=gitlab
+# TODO: where/how to get newrelic api key?
+# retrieve from somewhere on disk or a service?
+API_KEY=idunno
+
+if [[ $NEWRELIC_APP = snippets-dev-oregon-b ]]; then
+    K8S_NAMESPACE=snippets-dev
+elif [[ $NEWRELIC_APP = snippets-stage-oregon-b ]]; then
+    K8S_NAMESPACE=snippets-stage
+elif [[ $NEWRELIC_APP = snippets-prod-frankfurt ]]; then
+    K8S_NAMESPACE=snippets-prod
+fi
+
+# set the namespace before getting the deployment
+kubectl config set-context $(kubectl config current-context) --namespace=$K8S_NAMESPACE
+
+# get the image value from the base deployment matching the K8S_NAMESPACE
+FULL_IMAGE="kubectl get deploy ${K8S_NAMESPACE} -o jsonpath=\"{...image}\""
```

Comment:
This seems brittle; I believe it will break if we use any sidecar containers in the future.

Reply:
Would it be better to do the fully qualified path? Something like `$.spec.template.spec.containers[0].image`? But fine leaving as-is if it's too early for optimizing.